projects/igniteui-angular/src/lib/core/styles/components/button/_button-theme.scss

```diff
@@ -287,28 +317,35 @@
         indigo-design: $outlined-button-padding-indigo
     ), $variant);
 
+    $icon-button-width: map.get((
+        material: $icon-button-width-material,
+        fluent: $icon-button-width-fluent,
+        bootstrap: $icon-button-width-bootstrap,
+        indigo-design: $icon-button-width-indigo
+    ), $variant);
+
     $button--size-material: (
         comfortable: rem(36px),
         cosy: rem(30px),
         compact: rem(24px)
     );
 
     $button--size-fluent: (
-        comfortable: rem(32px),
-        cosy: rem(28px),
+        comfortable: rem(38px),
+        cosy: rem(32px),
         compact: rem(24px)
     );
 
     $button--size-bootstrap: (
         comfortable: rem(48px),
         cosy: rem(38px),
-        compact: rem(31px)
+        compact: rem(32px)
```

Comment:
@simeonoff and I discussed this matter and decided to move ahead with the 31px size, not because it is the correct design decision thing to do but because it is the only thing we can do to match Bootstrap design language I will log this as a task to enhance this in UI kits as we introduce button display densities